core/src/main/java/greencity/controller/EventController.java

```diff
@@ -168,13 +171,22 @@ public ResponseEntity<EventDto> getEvent(
     })
     @ApiPageableWithoutSort
     @GetMapping
-    public ResponseEntity<PageableAdvancedDto<EventDto>> getEvent(
+    public ResponseEntity<PageableAdvancedDto<EventDto>> getEvents(
         @Parameter(hidden = true) Pageable pageable,
         @RequestParam(required = false, name = "user-id") Long userId,
         FilterEventDto filterEventDto) {
```

Comment:
Please also look at FilterEventDto class, because it is expected that there will be List of EventTime (not just EventTime). Due to this problem event filter doesnt work on client side, when user selects several event times

Reply:
No. EventTime can be only FUTURE or PAST. I discuss this with Pavlo Poliak.